src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistry.scala

```diff
@@ -150,6 +154,19 @@ class WalletRegistry(store: LDBVersionedStore)(ws: WalletSettings) extends Score
       }
   }
 
+  def walletTxsBetween(heightFrom: Height, heightTo: Height): Seq[WalletTransaction] = {
+    val firstKey = firstIncludedScanTransactionSpaceKey(Constants.PaymentsScanId, heightFrom)
+    val lastKey = lastIncludedScanTransactionSpaceKey(Constants.PaymentsScanId, heightTo)
+
+    // Get wallet transactions from hightFrom (inclusive) to heightTo (inclusive)
+    val range = store.getRange(firstKey, lastKey)
+    range.flatMap { case (_, txId) =>
+      store.get(txKey(ModifierId @@ Base16.encode(txId))).flatMap { txBytes =>
+        WalletTransactionSerializer.parseBytesTry(txBytes).toOption
```

Comment:
I'd log error or warning in case of failed parsing if it is not expected ... it could save long hours of troubleshooting in case it ever happens ...

Reply:
Done!